tests/Model/Message/MessageBagTest.php

```diff
@@ -103,18 +103,23 @@ public function withoutSystemMessage(): void
         $messageBag = new MessageBag(
             Message::forSystem('My amazing system prompt.'),
             Message::ofAssistant('It is time to sleep.'),
+            Message::forSystem('A system prompt in the middle.'),
             Message::ofUser('Hello, world!'),
+            Message::forSystem('Another system prompt at the end'),
```

Comment:
i feel like we're running a bit in circles with this :laughing: in my understanding it is wrong to use multiple system/developer messages. can you point me again to your use-case so i can better understand what you're trying to achieve?

Reply:
hehe, just saying^^